docs/conf.py

```diff
@@ -95,7 +95,8 @@
 # General information about the project.
 project = 'MetPy'
 # noinspection PyShadowingBuiltins
-copyright = ('2019, MetPy Developers. Development supported by National Science Foundation grants '
+copyright = ('2019-2020, MetPy Developers. '
```

Comment:
Should probably be 2008 to match the top-level license.

Reply:
Done!